content/en/user-guide/aws/cloudwatch/index.md

```diff
@@ -7,14 +7,101 @@ aliases:
+            "metricFilterCount": 0,
+            "arn": "arn:aws:logs:us-east-1:000000000000:log-group:/aws/lambda/serverless-local-hello:*",
+            "storedBytes": 262
+        },
+        {
+            "logGroupName": "/aws/lambda/serverless-local-hello2",
+            "creationTime": 1683009865420,
+            "metricFilterCount": 0,
+            "arn": "arn:aws:logs:us-east-1:000000000000:log-group:/aws/lambda/serverless-local-hello2:*",
+            "storedBytes": 262
+        }
+    ]
+}
+{{< / command >}}
+
+Get the log streams for the Log Group using the [`DescribeLogStreams`](https://docs.aws.amazon.com/AmazonCloudWatchLogs/latest/APIReference/API_DescribeLogStreams.html) API. Run the following command to get the Log Stream name:
+
+{{< command >}}
+$ awslocal logs describe-log-streams --log-group-name /aws/lambda/serverless-local-hello
+{
+    "logStreams": [
+        {
+            "logStreamName": "2023/05/02/[$LATEST]853a59d0767cfaf10d6b29a6790d8b03",
+            "creationTime": 1683009968958,
+            "firstEventTimestamp": 1683009968920,
+            "lastEventTimestamp": 1683009968945,
+            "lastIngestionTime": 1683009968979,
+            "uploadSequenceToken": "1",
+            "arn": "arn:aws:logs:us-east-1:000000000000:log-group:/aws/lambda/serverless-local-hello:log-stream:2023/05/02/[$LATEST]853a59d0767cfaf10d6b29a6790d8b03",
+            "storedBytes": 262
+        }
+    ]
+}
+{{< / command >}}
+
+You can now fetch the log events using the [`GetLogEvents`](https://docs.aws.amazon.com/AmazonCloudWatchLogs/latest/APIReference/API_GetLogEvents.html) API. Run the following command to get the logs:
+
+{{< command >}}
+$ awslocal logs get-log-events --log-group-name '/aws/lambda/serverless-local-hello' --log-stream-name '2023/05/02/[$LATEST]853a59d0767cfaf10d6b29a6790d8b03'
+{
+    "events": [
+        {
+            "timestamp": 1683009968920,
+            "message": "START RequestId: 71712856-9f41-4d22-827c-e3883f799f25 Version: $LATEST",
+            "ingestionTime": 1683009968979
+        },
+        {
+            "timestamp": 1683009968932,
+            "message": "END RequestId: 71712856-9f41-4d22-827c-e3883f799f25",
+            "ingestionTime": 1683009968979
+        },
+        {
+            "timestamp": 1683009968945,
+            "message": "REPORT RequestId: 71712856-9f41-4d22-827c-e3883f799f25\tDuration: 1.27 ms\tBilled Duration: 2 ms\tMemory Size: 1024 MB\tMax Memory Used: 1024 MB\t",
+            "ingestionTime": 1683009968979
+        }
+    ],
+    "nextForwardToken": "f/00000000000000000000000000000000000000000000000000000002",
+    "nextBackwardToken": "b/00000000000000000000000000000000000000000000000000000000"
+}
+{{< / command >}}
+
+Optionally, you can use [filters](https://docs.aws.amazon.com/cli/latest/reference/logs/filter-log-events.html) or [queries](https://docs.aws.amazon.com/cli/latest/reference/logs/get-query-results.html) to refine your results.
```

Comment:
Filters are only supported in LocalStack Pro, maybe we should add this note here?